dh.sh

```diff
@@ -0,0 +1,185 @@
+externals="externals/dh-hdp-templates https://github.com/um-datahub/dh-hdp-templates.git
+externals/dh-hdp-zib-templates https://github.com/um-datahub/dh-hdp-zib-templates.git
+externals/dh-hdp-transform-rest https://github.com/MaastrichtUniversity/dh-hdp-transform-rest.git
+externals/dh-hdp-notebooks https://github.com/MaastrichtUniversity/dh-hdp-notebooks.git
+externals/dh-hdp-fhir-bridge https://github.com/MaastrichtUniversity/dh-hdp-fhir-bridge.git
+externals/dh-hdp-etl https://github.com/MaastrichtUniversity/dh-hdp-etl.git"
+
+
+is_local(){
+    if [[ $RIT_ENV == "local" ]]; then
+      return 0;
+    fi
+    return 1;
+
+}
+
+setup_requirements(){
+    echo -e "Update permissions of the folder filebeat/logs/ehrdb/"
+    mkdir -p ./filebeat/logs/ehrdb && chmod -R 777 ./filebeat/logs/ehrdb
+    mkdir -p ./filebeat/logs/ehrbase && chmod -R 777 ./filebeat/logs/ehrbase
+}
+
+dev_setup_requirements(){
+    if is_local; then
+      setup_requirements
+    fi
+}
+
+# do the required action in case of externals or exec
+if [[ $1 == "externals" ]]; then
+    action=${ARGS/$1/}
+    run_repo_action ${action} "${externals}"
+    exit 0
+fi
+
+if [[ $1 == "setup" ]]; then
+    setup_requirements
+    exit 0
+fi
+
+if [[ $1 == "transform" ]]; then
+    dev_setup_requirements
+    echo -e "\nStart Spring boot Rest API"
+    if is_local; then docker compose build transform-rest filebeat; fi
+    docker compose up -d transform-rest
+
+    echo -e "\nExit dh.sh"
+    exit 0
+fi
+
+if [[ $1 == "zib" ]]; then
+    dev_setup_requirements
+    if is_local; then docker compose build filebeat etl-zib transform-rest; fi
+
+    echo -e "\nRunning etl-zib"
+    docker compose up -d etl-zib
+    # Add a safe guard against infinite loop during a CI execution
+    SAFE_GUARD=0
+    until docker compose logs --tail 100 etl-zib 2>&1 | grep -q "Print all EHR ids available on the server";
+    do
+      if [[ $SAFE_GUARD -eq 15  ]]; then
+        echo -e "STOP waiting for etl-zib"
+        break
+      fi
+      ((SAFE_GUARD++))
+      echo -e "Waiting for etl-zib"
+      sleep 5
+    done
+
+    if [[ $SAFE_GUARD -ne 15  ]]; then
+      echo -e "\nPrint logs for etl-zib"
+      docker compose logs etl-zib
+      echo -e "\nExit dh.sh"
+      exit 0
+    else
+      echo -e "\nFailed to run etl-zib"
+      exit 1
+    fi
+fi
+
+if [[ $1 == "jupyter-zib" ]]; then
+    dev_setup_requirements
+    echo -e "\nExplore zib dataset"
+    if is_local; then docker compose build jupyter-zib transform-rest; fi
+    docker compose up -d jupyter-zib
+
+    echo -e "\nExit dh.sh"
+    exit 0
+fi
+
+if [[ $1 == "fhir" ]]; then
+    echo -e "\nStart FHIR Bridge"
+    docker compose up -d fhir-bridge
+    until docker compose logs --tail 100 fhir-bridge 2>&1 | grep -q "Started FhirBridgeApplication in";
+    do
+      echo -e "Waiting for FhirBridgeApplication"
+      sleep 10
+    done
+
+    echo -e "\nExit dh.sh"
+    exit 0
+fi
+
+if [[ $1 == "fhir-etl" ]]; then
```

Comment:
Regarding [DHHDP-483](https://mumc.atlassian.net/browse/DHHDP-483), "fhir-etl" can be removed?